iommu_data_structures.adoc

```diff
@@ -396,20 +398,23 @@ When `PDTV` is 0, the `DPE` bit is reserved for future standard extension. Until
 its use is defined, the bit should be cleared by software for forward
 compatibility, and must be ignored by hardware.
 
-The IOMMU supports the 1 setting of `GADE` and `SADE` bits if `capabilities.AMO`
-is 1. When `capabilities.AMO` is 0, these bits are reserved.
-
 If `GADE` is 1, the IOMMU updates A and D bits in G-stage PTEs atomically. If
-`GADE` is 0, the IOMMU behavior is determined by `fctl.ADFD` bit. If `fctl.ADFD`
-is 0 then the IOMMU causes a guest-page-fault. If `fctrl.ADFD` is 1, the IOMMU
-ignores the A and D bits in the PTEs; the IOMMU does not update the A or D bits
-and does not cause any faults based on A and/or D bit being 0.
+`GADE` is 0, the IOMMU behavior is determined by G-stage A/D fault enable -
+`GADFE` - control bit. If `GADFE` is 1 then the IOMMU causes a guest-page-fault.
+If `GADFE` is 0, the IOMMU ignores the A and D bits in the PTEs; the IOMMU does
+not update the A or D bits and does not cause any faults based on A and/or D
+bit being 0.
```

Comment:
If we go with this option, can we change the polarity. Make it GADFD (disable)
By default, IOMMU should raise page faults unless GADFD is set.